redCOMPONENT-COM/redSHOP
# REDSHOP-3068 Implement - frontend UI for redSHOP

I did:
- Add SCSS 
- Add bootstrap
- Change table tag to div tag
- Add billing and shipping layout to generic
- Update gulp.js
- REDSHOP-3074 Using Joomla Captcha plugin instead

How to test:
- Install redSHOP
- Create Category, Product, Manufacturer, Cart, Checkout page...
- Add to cart -> Checkout
- Check around to make sure all it's working
